test/extensions/filters/common/expr/context_test.cc

```diff
@@ -50,6 +51,17 @@ TEST(Context, InvalidRequest) {
   EXPECT_FALSE(header.has_value());
 }
 
+TEST(Context, InvalidRequestLegacy) {
+  TestScopedRuntime scoped_runtime;
+  scoped_runtime.mergeValues({{"envoy.reloadable_features.consistent_header_validation", "false"}});
+
+  Http::TestRequestHeaderMapImpl header_map{{"referer", "dogs.com"}};
+  Protobuf::Arena arena;
+  HeadersWrapper<Http::RequestHeaderMap> headers(arena, &header_map);
+  auto header = headers[CelValue::CreateStringView("dogs.com\n")];
```

Comment:
We probably should test a invalid header key/name in particular.

Reply:
so this does exercise the test code.  did you just prefer an invalid string that looks like a key? PTAL

Reply:
Yeah, i was preferring a invalid string that looks like a key. Thanks for updates.